pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -338,6 +340,14 @@ self: super: {
     dependencies = with self; [ nvim-yarp ];
   });
 
+  denops-vim = super.denops-vim.overrideAttrs (old: {
+    dependencies = [ deno ];
```

Comment:
I don't think this works, I get this message when I try to start up my neovim:
```console
[denops] A 'deno' (g:denops#deno) is not executable. Denops requires executable Deno.
```

Reply:
looking into that